docker/Dockerfile

```diff
@@ -38,4 +38,6 @@ USER zipkin
 
 EXPOSE 9410 9411
 
+HEALTHCHECK --interval=10s --start-period=30s --timeout=5s CMD wget --quiet http://localhost:9411/health || exit 1
```

Comment:
cc also @jcarres-mdsol in case you are using health checking in your deployment. This isn't the release dockerfile, but I suspect it will soon converge.